impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelInterpolator.java

```diff
@@ -137,7 +137,7 @@ String callback(
             String expression) {
         String value = doCallback(model, projectDir, request, problems, expression);
         if (value != null) {
-            // value = postProcess(projectDir, request, expression, value);
```

Comment:
- this looks like a real bug.
- test is green meaning blindspot, might use or loose.

Reply:
is this any good? @gnodet @elharo